lifetimes/estimation.py

```diff
@@ -699,8 +708,16 @@ def conditional_probability_alive(self, frequency, recency, T):
         Returns: a scalar
 
         """
+        ln_exp_max = 300.0
+
         r, alpha, a, b = self._unload_params('r', 'alpha', 'a', 'b')
-        return 1. / (1 + (frequency > 0) * (a / (b + frequency - 1)) * ((alpha + T) / (alpha + recency)) ** (r + frequency))
+        log_div = (r + frequency) * log(
```

Comment:
Nice fix @AdriMarteau! I was reading through the associated paper and wondering why the `log` were necessary. :-D